docs/analysis.rst

```diff
@@ -47,6 +47,20 @@ spectrum:
     <Quantity 9.84136331>
 
 
+A second method to calculate SNR does not require the uncertainty defined
+on the `~specutils.Spectrum1D` object. This computes the signal to noise 
+ratio DER_SNR following the definition set forth by the Spectral 
+Container Working Group of ST-ECF, MAST and CADC. This is based on the
```

Comment:
Can you add a link to the article about this (https://www.spacetelescope.org/static/archives/stecfnewsletters/pdf/hst_stecf_0042.pdf) somewhere? Right now it seems like it's an algorithm "out-of-nowhere", but that article pretty clearly outlines the motivation.

Maybe even pull a one-sentence (at most) description of the algorithm and/or its tradeoffs, derived from what's in that article?  (but maybe that's too hard to do, in which case just a reference to the article describing it is sufficient.)